src/tmpfiles/tmpfiles.c

```diff
@@ -1958,6 +1961,11 @@ static int parse_line(const char *fname, unsigned line, const char *buffer) {
         if (!should_include_path(i.path))
                 return 0;
 
+        if (!needs_glob(i.type) &&
+            chase_symlinks(i.path, NULL, CHASE_NO_AUTOFS, NULL) == -EREMOTE)
+                /* autofs automount point - not safe to touch */
+                return 0;
```

Comment:
I figure you meant to remove this bit?

Reply:
No, I left that deliberately.  You said "I am pretty sure we should do this check as part of the second step too."   "too" suggested to me that the current code should stay and an extra test should be added.  This made sense to me.
But I can remove that if you like - I'm happy either way.

Reply:
Please remove it. I mean "too" in the sense of "like the other checks", not as in "at both places"...

Sorry for the confusion